src/Riok.Mapperly/SymbolExtensions.cs

```diff
@@ -0,0 +1,62 @@
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+
+namespace Microsoft.CodeAnalysis;
+
+public static class SymbolExtensions
+{
+    public static bool IsPartialDefinition(this IMethodSymbol method)
+    {
+#if ROSLYN3_10_OR_GREATER
+        return method.IsPartialDefinition;
+#else
+        if (method.Locations.Any(loc => loc.IsInMetadata))
+        {
+            return false;
+        }
```

Comment:
This code base usually omits braces for single line if conditions with single line returns.